### PR TITLE
Portfolio: Desktop version

### PR DESCRIPTION
**Pull request by DamianDib9**
### **_Projects Requirements_**
___________________________________

- Use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px). ✔️
- In order to lay out the elements on the page you should use Grid and Flexbox. ✔️
- You don't need to implement any functionality that requires JavaScript. ✔️
- You must replicate the design as much as possible ✔️

Hope you like it! 😎👍

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -204,8 +243,8 @@ <h4>Skills</h5> <img class="arrowSep" src="./img/tech logos/arrow right.svg" alt
             </section>
 
 
-            <section class="contactMe">
-
+            <section id="contactMe" class="contactMe">
+<div id="contentContainer">
                 <h2 id="contactTitle">Contact me</h2>
                 <p class="aboutP" id="contactP">If you have an application you are interested in developing, a feature
                     that you need built or a project that needs coding.
```
Comment (Lizdev-05) [inline index.html:250]: - [x] Please make sure that you remove the vertical overflow of these elements in your contact section to make your design match the Figma design. :pray: 

  
  ### Your design 
  image
  
  ### Figma design
  image

**Review by Lizdev-05 — CHANGES_REQUESTED** [review]
Hi @DamianDib9  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Good PR :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please make sure that you remove the live demo section from your readme file since you have no link demo link yet :pray: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DamianDib9) [reply index.html:250]: image

OK! thanks for your kind review.
Fixed!

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @DamianDib9 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Your Project is well structured :white_check_mark: 
 * Good Job :+1:
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] Keep doing the Great work :clinking_glasses: 

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**